packages/web-app-files/src/components/AppBar/SelectedResources/BatchActions.vue

```diff
@@ -44,6 +39,11 @@ export default {
     Move,
     Restore
   ],
+  setup() {
+    return {
+      isFilesAppActive: useIsFilesAppActive()
```

Comment:
I'm confused, can't you delete this again with the helper function now being used in the mixins?